rocks/RocksDatabase.java

```diff
@@ -189,27 +190,22 @@ private void initialiseData() {
         } catch (RocksDBException e) {
             throw TypeDBException.of(e);
         }
+        mayInitRocksLogger();
     }
 
     protected void load() {
-        loadSchema();
+        initialiseOrLoadSchema();
+        validateEncodingVersion();
         loadData();
-        mayInitRocksLogger();
         isOpen.set(true);
         try (RocksSession.Schema session = createAndOpenSession(SCHEMA, new Options.Session()).asSchema()) {
             try (RocksTransaction.Schema txn = session.initialisationTransaction()) {
-                validateEncoding(txn.schemaStorage());
                 schemaKeyGenerator.sync(txn.schemaStorage());
                 dataKeyGenerator.sync(txn.schemaStorage(), txn.dataStorage());
             }
         }
     }
 
-    private void loadSchema() {
-        // loading a Rocks database with just the default column family is the same process are creating it
-        initialiseSchema();
-    }
-
```

Comment:
instead of an empty method with a comment, we update the naming to `initialiseOrLoadSchema()`